core/templates/dev/head/pages/signup/redirect_to_login_directive.html

```diff
@@ -0,0 +1,11 @@
+<div class="modal-header">
+  <h3>Redirect to Login Page</h3>
+</div>
+
+<div class="modal-body">
+  <p>Sorry, you have been logged out [probably in another window]. To log in again please click on the redirect to login button.</p>
```

Comment:
Make this a translatable constant. Also, change text to:

`Sorry, your registration session has expired. Please click "Continue Registration" to restart the process.`

and change the button to "Continue Registration".

Reply:
Done.